diffkemp/simpll/passes/FunctionAbstractionsGenerator.cpp

```diff
@@ -26,6 +26,9 @@ AnalysisKey FunctionAbstractionsGenerator::Key;
 /// and for each pair of assembly code and constraint.
 FunctionAbstractionsGenerator::Result FunctionAbstractionsGenerator::run(
         Module &Mod, AnalysisManager<Module, Function *> &mam, Function *Main) {
+    DEBUG_WITH_TYPE(DEBUG_SIMPLL,
+                    dbgs() << "Generating function abstractions in module "
```

Comment:
```suggestion
                    dbgs() << "Generating function abstractions in "
```
... same for other messages (the word "module" is redundant here).